src/model/bpmn/internal/types.ts

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2023 Bonitasoft S.A.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * @internal
+ */
+export type ShapeColorExtensions = 'bv:color:fill' | EdgeColorExtensions;
```

Comment:
**question:** 
What is the purpose or meaning of the 'bv:' prefix in the property name? Since we're working within the internal model, it's not clear to me why it's necessary or how it adds value. 
Could you please provide some clarification or explanation as to its usefulness?

Reply:
See explanations in the comment above